config/default.yml

```diff
@@ -127,6 +127,11 @@ Minitest/EmptyLineBeforeAssertionMethods:
   Enabled: pending
   VersionAdded: '0.23'
 
+Minitest/FileName:
+  Description: 'Checks if test file names start with `test_` or end with `_test.rb`.'
```

Comment:
It would be nice to add a style guide link, when it will be merged - https://github.com/rubocop/rubocop-minitest/issues/213#issuecomment-1368029199